apps/explorer/src/pages/validator/DelegationAmount.tsx

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { SUI_TYPE_ARG } from '@mysten/sui.js';
+
+import { CoinFormat, useFormatCoin } from '~/hooks/useFormatCoin';
+import { Heading } from '~/ui/Heading';
+import { Text } from '~/ui/Text';
+
+type DelegationAmountProps = {
+    amount: bigint | number | string;
+    isStats?: boolean;
+};
+
+export function DelegationAmount({ amount, isStats }: DelegationAmountProps) {
```

Comment:
Sorry that I'm giving this feedback now, missing this in earlier review, but these shouldn't be in `pages/`. That directory should ideally roughly mirror our routing structure (there's still some work to do to make that happen). Instead these really should be somewhere in `components/`